product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,265 @@
+---
+
+## Installing the PGD software
+
+With the repositories configured, you can now install the Postgres Distributed software. 
+These steps must be carried out on each host.
+
+* Install the PGD packages which include a server specific BDR package and generic PGD proxy and cli packages.
+* Initialise the Postgres database.
+* Add the BDR extension to the shared_preload_libraries setting in `postgresql.conf`.
+* Set the `wal_level` GUC variable to `logical` in `postgresql.conf`.
+* Turn on commit timestamp tracking by setting `track_commit_timestamp` to `'on'` in `postgresql.conf`.
+* Set a password on the EnterprisedDB/Postgres user.
+* Raise the maximum worker processes to 16 by setting `max_worker_processes` to `'16'` in `postgresql.conf`.
+* Add rules to `pg_hba.conf` to allow nodes to connect to each other.
+* Add a `.pgpass` file to allow nodes to authenticate each other.
+* Restart the server.
+* Log into the server's default database (`ebd` for EPAS, `postgres` for PGE and Community).
+* Use `CREATE DATABASE bdrdb` to create the default PGD replicated database.
+* Log out and then log back in to `bdrdb`.
+* Use `CREATE EXTENSION bdr` to enable the BDR extension and PGD to run on that database.
+
+We will look in detail the steps for EDB Postgres Advanced Server in the worked example below. 
+
+If you are installing PGD with EDB Postgres Extended Server or Community Postgres, the steps are similar, but details such as package names and paths are different. These differences are summarized in [Installing PGD for EDB Postgres Extended Server](#installing_pgd_for_edb_postgres_extended_server) and [Installing PGD for Community Postgresql](#installing_pgd_for_community_postgresql).
+
+## Worked example
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+For example, if we are installing EDB Postgres Advanced Server (epas) version 16, we would install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: `edb-pgd5-proxy` for PGD Proxy and `edb-pgd5-cli` for the PGD command line tool.
+
+To install all of these packages on a RHEL or RHEL compatible Linux, run:
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+```
+
+### Initializing and starting the database
+
+Now initialise the database. You will need to know the path to the setup script for your particular Postgres. 
+For EDB Postgres Advanced Server, this script can be found in `/usr/edb/as16/bin` as `edb-as-16-setup`. 
+This command needs to be run with the `initdb` parameter and we need to pass an option setting the database to use UTF-8.
+
+
+```
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+```
+
+Once the database is initialised, we will start it which will enable us to continue configuring the BDR extension.
+
+```
+sudo systemctl start edb-as-16
+```
+
+### Configuring the BDR extension
+
```

Comment:
As someone who is familiar with configuring PostgreSQL (which is will be a characteristic of at least part of the audience for this document) I find this section very difficult to read (especially as the example commands render as black sections with long lines requiring horizontal scrolling).

Personally I'd rather see a plain description of each required GUC, possibly as a table, and possibly including links to each GUC in the PostgreSQL documentation, then I can get a quick easy overview of the required GUCs without having to squint through shell commands which I might or might not use.

The shell code for adding the GUCs for each PG variant is in the summary section anyway, for anyone who needs a full set of instructions (which may be another segment of audience for this document).

Reply:
The section now opens with this summary:

## Installing the PGD software

With the repositories configured, you can now install the Postgres Distributed software. 
These steps must be carried out on each host.

* Install the PGD packages which include a server specific BDR package and generic PGD proxy and cli packages.
* Initialise the Postgres database.
* Add the BDR extension to the shared_preload_libraries setting in `postgresql.conf`.
* Set the `wal_level` GUC variable to `logical` in `postgresql.conf`.
* Turn on commit timestamp tracking by setting `track_commit_timestamp` to `'on'` in `postgresql.conf`.
* Raise the maximum worker processes to 16 by setting `max_worker_processes` to `'16'` in `postgresql.conf`.
* Set a password on the EnterprisedDB/Postgres user.
* Add rules to `pg_hba.conf` to allow nodes to connect to each other.
* Add a `.pgpass` file to allow nodes to authenticate each other.
* Restart the server.
* Log into the server's default database (`ebd` for EPAS, `postgres` for PGE and Community).
* Use `CREATE DATABASE bdrdb` to create the default PGD replicated database.
* Log out and then log back in to `bdrdb`.
* Use `CREATE EXTENSION bdr` to enable the BDR extension and PGD to run on that database.

which should be as little as possible for an experienced user to use.